src/onResize.js

```diff
@@ -54,26 +54,26 @@ export default function onResize(){
             highlight(id);
         });
 
-    this.svg.selectAll(".point")
-        .on("mouseover",function(d){ 
+    this.svg.selectAll('.point')
```

Comment:
Clicking a line gives the expected behaviour, but clicking a point (that is a <circle>) doesn't seem to leave the participant highlighted in the main chart. Expect there just needs to be a copy/paste update in this area.